e2e/pipelines/test_standard_pipelines_batch.py

```diff
@@ -0,0 +1,106 @@
+from haystack.document_stores import InMemoryDocumentStore
+from haystack.pipelines import FAQPipeline, DocumentSearchPipeline, MostSimilarDocumentsPipeline
+from haystack.nodes import EmbeddingRetriever
+from haystack.schema import Document
+
+
+def test_faq_pipeline_batch():
+    documents = [
+        {"content": "How to test module-1?", "meta": {"source": "wiki1", "answer": "Using tests for module-1"}},
+        {"content": "How to test module-2?", "meta": {"source": "wiki2", "answer": "Using tests for module-2"}},
+        {"content": "How to test module-3?", "meta": {"source": "wiki3", "answer": "Using tests for module-3"}},
+        {"content": "How to test module-4?", "meta": {"source": "wiki4", "answer": "Using tests for module-4"}},
+        {"content": "How to test module-5?", "meta": {"source": "wiki5", "answer": "Using tests for module-5"}},
+    ]
+    document_store = InMemoryDocumentStore()
+    retriever = EmbeddingRetriever(document_store=document_store, embedding_model="deepset/sentence_bert")
```

Comment:
A smaller and thus faster model alternative is `sentence-transformers/all-MiniLM-L6-v2`. Could replace `deepset/sentence_bert` everywhere. It's already used in a few other tests and is also used in the faq example: examples/basic_faq_pipeline.py